### PR TITLE
Natural systems data sourcing

### PR DESCRIPTION
**Pull request by pawilfahrt**
## Checklist

Please complete this checklist as a courtesy to the PR reviewer.  

Code and styling
* [x] All of the files/scripts I added are in the right place and named appropriately. See the [README](https://github.com/Metropolitan-Council/ghg-cprg/blob/main/README.md) for details.
* [x] I have not used `setwd()`
* [x] I have used `file.path(here::here(), "file_name"))` to source any scripts or read in data
* [x] I have not added any large datasets, unless absolutely necessary (explain)
* [x] I have commented my code, particularly in hard to understand areas
* [x] I have added additional package dependencies as necessary with `renv::install()`
* [x] I have run `styler::style_dir(".", recursive = TRUE, filetype = c("R", "qmd"))`
* [x] Plots
  - [x] If plotly, use `source = "chunk-name"` in `plot_ly()`
  - [x] Use formatting with `councilR::plotly_layout()`?
* [x] Chunk formatting
  - [x] All chunks named
  - [x] All figure or table chunks have caption
  - [x] `out.width: "95%"`. If a specific height is needed, use pixels. and/or `out.height: "500px"`

Document editing
* [x] I have ensured that modified documents knit successfully from [`render_for_publication.R`](https://github.com/Metropolitan-Council/ghg-cprg/blob/main/R/render_for_publication.R)
* [x] I have fixed any missing citations, cross references, hyperlinks
* [x] I have reviewed my contributions for typos and misspellings.

GitHub and project management
* [x] I have identified and assigned at least one Reviewer (Liz, Sam, Laine) to this PR
* [x] I have assigned myself to this PR
* [ ] I have updated the status in the [GitHub Project](https://github.com/orgs/Metropolitan-Council/projects/8/views/1)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -325,3 +325,4 @@ _nature/data-raw/land_cover_stock.csv
 /_nature/_natural_systems_files
 
 site_libs/
+*.csv
```
Comment (eroten) [inline .gitignore:328]: This is probably too broad for the entire repository. Can you gitignore the individual CSVs instead?

File: _nature/county_land_cover_sequestration_ns.R [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+
+saveRDS(wc_county_c, "./_nature/data/county_landcover_sequestration_2021.rds")
+saveRDS(wc_county_c_meta, "./_nature/data/county_landcover_sequestration_2021_meta.rds")
+
+county_seq_total <- summarize(wc_county_c, seq_total = sum(sequestration_potential))
+county_stock_total <- summarize(wc_county_c, stock_total = sum(stock_potential))
+
+
+sum(county_seq_total$seq_total) # -2180841 ! Liz gets 2390357
```
Comment (eroten) [inline _nature/county_land_cover_sequestration_ns.R:33]: @pawilfahrt  can you double check this value? I'm getting a slightly different one than is commented here, but the overall numbers rendered in Quarto match regardless.

Comment (pawilfahrt) [reply _nature/county_land_cover_sequestration_ns.R:33]: This was an internal check as I built the code up, I likely didn't update after a small tweak to a sequestration rate value. I also now get the value you do and will delete this check as there is now a test that accomplishes the same.

**Review by eroten — APPROVED** [review]
Whew! Thanks for all your work, Peter! It was fun to collaborate with you on this PR.

**Pull request merged**